packages/editor/src/lib/editor/managers/FocusManager.ts

```diff
@@ -20,7 +20,6 @@ export class FocusManager {
 			'instance',
 			(prev, next) => {
 				if (prev.isFocused !== next.isFocused) {
-					next.isFocused ? this.focus() : this.blur()
```

Comment:
so, this change was reworked in https://github.com/tldraw/tldraw/pull/3718 
but it's been like this for a long while: https://github.com/tldraw/tldraw/blob/79460cbf3a1084ac5b49e41d1e2570e4eee98e82/packages/tldraw/src/lib/defaultSideEffects.ts
the logic above was moved to FocusManager in #3718

what is this trying to fix?

Reply:
how it was before made it impossible to toggle the editor's focused state _without_ triggering a focus or blur event on the container. This is bad when tldraw is being used on a scrollable page (e.g. notion) because .focus triggers a scroll event.

We already added the editor.focus and editor.blur methods for toggling those states, and we also added options to those methods to disable the dom focus/blur events, but those options were effectively not being respected because of this line.

Reply:
makes sense. let's just be sure to communicate this out as a breaking change (as you note in the desc.)